src/ResourceManager/Cdn/AzureRM.Cdn.psd1

```diff
@@ -9,7 +9,7 @@
 @{  
   
 # Version number of this module.  
-ModuleVersion = '1.0.5' 
```

Comment:
please revert this change. It is part of our process to update versions in each release.